src/audio-listener.ts

```diff
@@ -44,7 +45,27 @@ export class AudioListener extends Component {
      */
     private time: number = 0;
 
-    update(dt: number) {
+    start() {
+        try{
+            /* Test if the recommended WebAudio function is available */
+            this.listener.positionX.linearRampToValueAtTime(tempVec[0], this.time);
```

Comment:
I recommend checking for the function with `if` instead, if that's possible.

```ts
if(typeof(this.listener.positionX.linearRampToValueAtTime) !== 'undefined') /* supported */
else /* unsupported */
```

Reply:
That didnt work, but i found a way